README.md

```diff
@@ -91,12 +96,19 @@ matches the constraints from the manifest. If the dependency is missing from
 
 ### Adding a dependency
 
-1. `import` the package in your `*.go` source code file(s).
-1. Run the following command to update your `Gopkg.lock` and populate `vendor/` with the new dependency.
+Adding a project as a dependency has three essential steps:
 
-    ```sh
-    $ dep ensure
-    ```
+1. `import` a package from the project in one of your `*.go` source files
+2. Add a version constraint on the project to `Gopkg.toml` (Optional, but recommended)
+3. Run `dep ensure`
+
+`dep ensure -add` provides some CLI sugar to ease this process:
```

Comment:
Since ensure is in a half implemented state, we should say exactly what the "sugar" is and it replaces all of steps 1-3, e.g. Alternatively you can run `dep ensure -add` to add the project to Gopkg.lock and your vendor directory, then you can immediately start using the project in your code.